### ISSUE TITLE
about training

### ISSUE THREAD

**Issue by xiongzhu666** [xuebinqin/U-2-Net#227]
Hi~Mr Qin，I am coming again....
I retrain u2net on my datasets（9000 pics），the loss value oscillate around 0.2，do you think should I keep training？？（I just train for 1.5 day），I am a little worried that keep training will cause overfitting...

**Comment by 1187128861** [comment]
> Hi~Mr Qin，I am coming again....
> I retrain u2net on my datasets（9000 pics），the loss value oscillate around 0.2，do you think should I keep training？？（I just train for 1.5 day），I am a little worried that keep training will cause overfitting...

你好，请问你的数据集是怎么制作的，可以的话麻烦告知下：Q-1187128861
